### PR TITLE
Add GP support in xtest build

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

**Comment by pascal-brand-st-dev** [comment]
@jbech-linaro Could you have a look at it?

File: qemu.mk [hunk 1]
```diff
@@ -165,9 +165,37 @@ xtest: optee-os optee-client
 		CROSS_COMPILE_HOST="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
 		CROSS_COMPILE_TA="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
 		TA_DEV_KIT_DIR=$(OPTEE_OS_PATH)/out/arm-plat-vexpress/export-user_ta \
+		CFG_ARM32=y \
+		CFG_DEV_PATH=/home/brand/devel/optee \
```
Comment (jbech-linaro) [inline qemu.mk]: This needs to be updated to something "non-branded" ;-)

File: qemu.mk [hunk 2]
```diff
@@ -165,9 +165,37 @@ xtest: optee-os optee-client
...
 
+xtest-clean:
+	@if [ -d "$(OPTEE_TEST_PATH)" ]; then \
+		make -C $(OPTEE_TEST_PATH) \
+		-j`getconf _NPROCESSORS_ONLN` \
+		CROSS_COMPILE_HOST="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
+		CROSS_COMPILE_TA="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
+		TA_DEV_KIT_DIR=$(OPTEE_OS_PATH)/out/arm-plat-vexpress/export-user_ta \
+		CFG_ARM32=y \
+		CFG_DEV_PATH=/home/brand/devel/optee \
```
Comment (jbech-linaro) [inline qemu.mk]: Same here ...

File: qemu.mk [hunk 3]
```diff
@@ -165,9 +165,37 @@ xtest: optee-os optee-client
...
+
+xtest-patch: optee-os optee-client
+	@if [ -d "$(OPTEE_TEST_PATH)" ]; then \
+		make -C $(OPTEE_TEST_PATH) \
+		-j`getconf _NPROCESSORS_ONLN` \
+		CROSS_COMPILE_HOST="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
+		CROSS_COMPILE_TA="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
+		TA_DEV_KIT_DIR=$(OPTEE_OS_PATH)/out/arm-plat-vexpress/export-user_ta \
+		CFG_ARM32=y \
+		CFG_DEV_PATH=/home/brand/devel/optee \
```
Comment (jbech-linaro) [inline qemu.mk]: And here ..

**Comment by jbech-linaro** [comment]
When `.../brand/...` has been replaced:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by pascal-brand-st-dev** [comment]
Right. I made an update

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Pull request merged**